server_extensions_extra/build.gradle

```diff
@@ -71,6 +71,22 @@ dependencies {
     githubCompile(group: 'com.github.GENERALBYTESCOM.bitrafael_public', name: 'bitrafael-client', version: bitrafaelVersion) {
         exclude group: 'org.slf4j', module: 'slf4j-simple'
     }
+	compile (group: 'org.bitcoinj', name:'bitcoinj-core', version: '0.14.7')
```

Comment:
bitcoinj should not be used. It will conflict with the rest of server server code.

Reply:
bitcoinj depency is needed by aepp-sdk-java 1.2.0. Can you point version which wouldnt conflict ?